tests/test_hecuba.py

```diff
@@ -0,0 +1,193 @@
+import gc
+import unittest
+
+import numpy as np
+from hecuba import config
+from pycompss.api.api import compss_wait_on
+from sklearn.datasets import make_blobs
+
+import dislib as ds
+from dislib.cluster import KMeans
+from dislib.decomposition import PCA
+from dislib.neighbors import NearestNeighbors
+from dislib.regression import LinearRegression
+
+
+class HecubaTest(unittest.TestCase):
```

Comment:
Can you also add tests for the different types of slicing and iterating through columns?

Reply:
* Added test_get_slice_dense just like in test_array.py.
* _merge_blocks is called by Array.collect(), I have several tests that use that method. If you want I can create another test using _merge_blocks specifically.
* As it is done right now, we cannot iterate through columns unless it is decided when initializing the Hecuba array. It is not used in any algorithm for now, so it shouldn't be a problem.